doc/DeepSpeech.rst

```diff
@@ -0,0 +1,76 @@
+Introduction
+============
+
+In this project we will reproduce the results of
+`Deep Speech: Scaling up end-to-end speech recognition <http://arxiv.org/abs/1412.5567>`_.
+The core of the system is a bidirectional recurrent neural network (BRNN)
+trained to ingest speech spectrograms and generate English text transcriptions.
+
+Let a single utterance :math:`x` and label :math:`y` be sampled from a training set
+`S = \{(x^{(1)}, y^{(1)}), (x^{(2)}, y^{(2)}), . . .\}`.
```

Comment:
That was my mistake, looks like I missed all the equations that start in the beginning of the line when converting the document. I fixed it.